global.json

```diff
@@ -1,13 +1,14 @@
 {
   "tools": {
-    "dotnet": "3.1.200",
+    "dotnet": "5.0.100-preview.8.20417.9",
     "vs": {
-      "version": "16.6"
+      "version": "16.9"
```

Comment:
Sorry, I don't understand but what does VS 16.9 version mean, that is the next version?

Reply:
stolen from [here](https://github.com/dotnet/roslyn/pull/47165#discussion_r478573904) I wanted to see if this passed the build first. I am not convinced this is actually what we want. Will need to confirm this doesn't break external contributors.

Reply:
alright, talked to the arcade folks. It just needs to be newer that preview 6. I am going to update us to be on preview 7